src/compiler/core/src/commonTest/kotlin/community/flock/wirespec/compiler/core/CompileFullEndpointTest.kt

```diff
@@ -526,6 +527,25 @@ class CompileFullEndpointTest {
         compiler { TypeScriptEmitter() } shouldBeRight ts
     }
 
+    @Test
+    fun python() {
+        val python = """
+            |from abc import abstractmethod
+            |from dataclasses import dataclass
+            |from typing import List, Optional
+            |
+            |from .shared.Wirespec import T, Wirespec
+            |
+            |@dataclass
+            |class PotentialTodoDto:
+            |  name: 'str'
+            |  done: 'bool'
+            |
+            |
+        """.trimMargin()
+        compiler { PythonEmitter() } shouldBeRight python
+    }
```

Comment:
Given the the `compiler` includes an endpoint , I'm surprised to not see that here. This seems somewhat counterintuitive.

```
            endpoint PutTodo PUT PotentialTodoDto /todos/{id: String} ? {done: Boolean} -> {
              200 -> TodoDto
              201 -> TodoDto
              500 -> Error
            }
```

Reply:
Python splits the code over multiple files. This is only the first file. I have added an endpoint file.